drivers/lora/sx12xx_common.c

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) 2019 Manivannan Sadhasivam
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <drivers/lora.h>
+#include <logging/log.h>
+#include <zephyr.h>
+
+/* LoRaMac-node specific includes */
+#include <radio.h>
+
+LOG_MODULE_REGISTER(sx12xx_common, CONFIG_LORA_LOG_LEVEL);
+
+static struct sx1276_data {
```

Comment:
This structure name should be sx12xx_data.

Reply:
ack